server/tests/cli/create-transcoding-job.ts

```diff
@@ -100,6 +102,32 @@ describe('Test create transcoding jobs', function () {
     }
   })
 
+  it('Should run a transcoding job on video 1 with resolution', async function () {
+    this.timeout(60000)
+
+    const env = getEnvCli(servers[0])
+    await execCLI(`${env} npm run create-transcoding-job -- -v ${video1UUID} -r 480`)
+
+    await waitJobs(servers)
+
+    for (const server of servers) {
+      const res = await getVideosList(server.url)
+      const videos = res.body.data
+      expect(videos).to.have.lengthOf(2)
+
+      const video = videos.find(v => v.uuid === video1UUID)
```

Comment:
What's the point of this?

Reply:
Powerful rhetorical question. This line is removed :).